sample/build.gradle

```diff
@@ -2,34 +2,35 @@ apply plugin: 'com.android.application'
 apply from: '../config/android-quality.gradle'
 
 android {
-    compileSdkVersion 25
-    buildToolsVersion "25.0.1"
-    defaultConfig {
-        applicationId "com.schibsted.spain.barista.sample"
-        minSdkVersion 9
-        targetSdkVersion 25
-        versionCode 1
-        versionName "1.0"
-        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
-    }
-    buildTypes {
-        release {
-            minifyEnabled false
-            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
-        }
+  compileSdkVersion 25
+  buildToolsVersion "25.0.1"
+  defaultConfig {
+    applicationId "com.schibsted.spain.barista.sample"
+    minSdkVersion 9
+    targetSdkVersion 25
+    versionCode 1
+    versionName "1.0"
+    testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+  }
+  buildTypes {
+    release {
+      minifyEnabled false
+      proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
     }
+  }
 }
 
 dependencies {
-    compile fileTree(dir: 'libs', include: ['*.jar'])
-    compile 'com.android.support:appcompat-v7:25.1.0'
-    compile 'com.android.support:recyclerview-v7:25.1.0'
-    compile "com.android.support:design:25.1.0"
+  compile fileTree(dir: 'libs', include: ['*.jar'])
 
-    testCompile 'junit:junit:4.12'
 
-    androidTestCompile project(':library')
-    androidTestCompile 'com.android.support.test.espresso:espresso-core:2.2.2'
-    androidTestCompile 'com.android.support:recyclerview-v7:25.1.0'
+  androidTestCompile project(':library')
 
+  compile 'com.android.support:appcompat-v7:25.1.0'
+  compile 'com.android.support:recyclerview-v7:25.1.0'
+  compile 'com.android.support:design:25.1.0'
+  compile 'com.android.support.constraint:constraint-layout:1.0.2'
+  testCompile 'junit:junit:4.12'
+  androidTestCompile 'com.android.support.test.espresso:espresso-core:2.2.2'
+  androidTestCompile 'com.android.support:recyclerview-v7:25.1.0'
```

Comment:
If you can leave this file as it was formatted, it will be better, as all `androidTestCompile` lines will appear together. But no problem if it's a problem, we can do it later :·)